ShaderConverter/ShaderConv/context.cpp

```diff
@@ -2315,13 +2315,7 @@ CContext::Translate_TEX( const CInstr& instr )
         {
             if (TextureUsesHardwareShadowMapping(dwStage))
             {
-                //Not sure if they can do a hardware shadow map with bias..
-                //Emit Magenta error color: mov dest 1 0 1 1
-                Modifiers modifiers = Modifiers();
-                this->EmitDstInstruction( modifiers,
-                                          D3D10_SB_OPCODE_MOV,
-                                          dest,
-                                          COperand( 1.0f, 0.0f, 1.0f, 1.0f ) );
+                EmitSampleCmp(instr, dest, dwStage, maskedSrc0);

```

Comment:
Just to confirm, the intention is that the bias is ignored and it's treated as a normal shadow compare? Last time we saw this, it was an app bug leaving a shadow texture bound for the wrong shader. Sounds like this is actually affecting shadows though.